src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,8 +1,28 @@
-export const TodoFilter = () => (
+import { Dispatch, SetStateAction } from 'react';
+
+interface Props {
+  query: string;
+  setQuery: Dispatch<SetStateAction<string>>;
```

Comment:
try not to expose more than you need

```suggestion
  setQuery: (query: string) => void
```